### PR TITLE
add relinkdxyz to showcase.json

### PR DESCRIPTION
**Pull request by luckerninja**

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @luckerninja,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/445) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: showcase.json [hunk 1]
```diff
@@ -3516,5 +3516,24 @@
...
   "youtube": "",
   "submittableId": ""
+  },
+  {
+  "id": "relinkdxyz",
+  "name": "{r}elinkd",
+  "oneLiner": "Digital identity + scores protocol and dapp", 
+  "website": "https://relinkd.xyz/",
+  "tags": [
+    "DID",
```
Comment (jennifertrin) [inline showcase.json:3526]: Can you please check this list for the allowed tags? https://github.com/dfinity/portal?tab=readme-ov-file#tags

I believe `Tools/Infrastructure` is the best tag for {r}elinkd.

**Review by jennifertrin — CHANGES_REQUESTED** [review]
Can you please also follow the CLA steps?

File: showcase.json (hunk 1 above)
Comment (luckerninja) [reply showcase.json]: thanks for the information, updated tags

**Review by jennifertrin — DISMISSED** [review]

**Comment by jennifertrin** [comment]
> Can you please also follow the CLA steps?

@L

> Dear @luckerninja,
> 
> In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).
> 
> If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/445) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.
> 
> — The DFINITY Foundation

@luckerninja Can you please also go through these CLA steps?

File: showcase.json (hunk 1 above)
Comment (jennifertrin) [reply showcase.json]: Thank you so much!

**Comment by luckerninja** [comment]
@jennifertrin, sure, I have confirmed the CLA - https://github.com/dfinity/cla/issues/445

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**